lib/awful/widget/tasklist.lua

```diff
@@ -672,6 +678,15 @@ function tasklist.filter.focused(c, screen)
     return get_screen(c.screen) == get_screen(screen) and capi.client.focus == c
 end
 
+--- Get all the clients in an undefined order.
```

Comment:
The order is not really undefined. Dunno, if that's worth mentioning.
This is "the" order of clients. By default, this is the order in which they were created, but `:swap` modifies this order. This is e.g. also the order according to which the "next" client is defined.

Reply:
/me Back to civilization, sorry for the AFK week.

> Dunno, if that's worth mentioning.

I don't think so. I used "undefined" instead of "random" on purpose. To me, the order is an implementation detail and it doesn't make much sense on its own.

Reply:
Well... I approved this a week ago. So I guess, a week ago I thought this was good to go. That's good enough for me so that I don't take a second look.

Off-topic: I'll be AFK next week